keras_nlp/models/roberta/roberta_preprocessing_test.py

```diff
@@ -0,0 +1,152 @@
+# Copyright 2022 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for RoBERTa preprocessing layers."""
+
+import os
+
+import tensorflow as tf
+from absl.testing import parameterized
+from tensorflow import keras
+
+from keras_nlp.models.roberta.roberta_preprocessing import RobertaPreprocessor
+
+
+class RobertaPreprocessorTest(tf.test.TestCase, parameterized.TestCase):
+    def setUp(self):
+        vocab = {
+            "<s>": 0,
+            "<pad>": 1,
+            "</s>": 2,
+            "Ġair": 3,
+            "plane": 4,
+            "Ġat": 5,
+            "port": 6,
+            "Ġkoh": 7,
+            "li": 8,
+            "Ġis": 9,
+            "Ġthe": 10,
+            "Ġbest": 11,
+        }
+
+        merges = ["Ġ a", "Ġ t", "Ġ k", "Ġ i", "Ġ b", "Ġa i", "p l", "n e"]
+        merges += ["Ġa t", "p o", "r t", "o h", "l i", "Ġi s", "Ġb e", "s t"]
+        merges += ["Ġt h", "Ġai r", "pl a", "Ġk oh", "Ġth e", "Ġbe st", "po rt"]
+        merges += ["pla ne"]
+
+        self.preprocessor = RobertaPreprocessor(
+            vocabulary=vocab,
+            merges=merges,
+            sequence_length=12,
+        )
+
+    def test_tokenize(self):
+        input_data = [" airplane at airport"]
+
+        output = self.preprocessor(input_data)
+        self.assertAllEqual(
+            output["token_ids"], [0, 3, 4, 5, 3, 6, 2, 1, 1, 1, 1, 1]
+        )
+        self.assertAllEqual(
+            output["padding_mask"], [1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0]
+        )
+
+    def test_tokenize_batch(self):
+        input_data = tf.constant(
+            [
+                " airplane at airport",
+                " airplane at airport",
+                " airplane at airport",
+                " airplane at airport",
+            ]
+        )
+
+        output = self.preprocessor(input_data)
+        self.assertAllEqual(
+            output["token_ids"],
+            [[0, 3, 4, 5, 3, 6, 2, 1, 1, 1, 1, 1]] * 4,
+        )
+
+        self.assertAllEqual(
+            output["padding_mask"], [[1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0]] * 4
+        )
+
+    def test_tokenize_multiple_sentences(self):
+        sentence_one = " airplane at airport"
+        sentence_two = " kohli is the best"
```

Comment:
Did we want to keep our examples lame @mattdangerw? Not sure if you weighed in on that discussion

Reply:
Oh I'm down for fun examples! Without a terribly strong opinion here.

We should make sure to follow https://opensource.google/conduct/, and trashing someone's favorite sports team etc. might not be the best.

But stuff like this sounds good to me! We want a fun contributing culture.